README.md

```diff
@@ -137,6 +138,36 @@ The available flags are:
   - `add-module-exports` toggles whether the UMD output of ez-build should be backwards compatible with AMD and CJS module formats. If this flag is specified, ez-build will ensure any module with a single `export default` will not export an object with a `default` key. This flag is disabled by default. It is only recommended you use this flag if you *must* keep backwards compatibility with legacy code.
   - `es2017` toggles support for compiling code ES2017 code. When ES2017 is ratified this flag will be removed and the ES2017 preset will be added by default. This flag is diabled by default.
 
+### `@<path>`
+
+Reads ez-build options from the file at `<path>`, resolved from the current working directory. This can be used to share configuration among different builds. For instance, a common scenario is to share the same build configuration between `--production` and `--interactive` builds in npm scripts, like so:
+
+```json
+{
+  "build": "ez-build --production --include \"js:**/*.js,js:**/*.jsx\" --flags es2017",
```

Comment:
Good point re: globs! Unfortunately, due to how ez-build parses this flag, it probably won't work even though it *does* support globs. I'd consider this a bug, and we should fix it, because you're right – this should be supported.

Regarding your second statement about a `js` folder – the `js:` part doesn't actually refer to any kind of directory structure. It's just a label for the JS pipeline, so `--include "js:**/*.{js,jsx}"` just means "for the JS pipeline, include these files in the build process." By default, ez-build assumes sources are found under `src/`, but this can be overridden by the `-i|--src` flag or by setting the `directories` key in package.json. The [readme has more info on this](https://github.com/zambezi/ez-build/blob/4427125078dddd1cc67e90f634a217dd412c6272/README.md#-i---src-dir) as well.